flexmeasures/ui/templates/crud/asset.html

```diff
@@ -274,42 +274,18 @@ <h3>Edit {{ asset.name }}</h3>
             </div>
             <div class="sensors-asset card">
                 <h3>All sensors for {{ asset.name }}</h3>
+                
                 <div class="table-responsive">
-                    <table class="table table-striped paginate nav-on-click" title="View data">
-                        <thead>
-                            <tr>
-                                <th>Name</th>
-                                <th>ID</th>
-                                <th class="text-right">Unit</th>
-                                <th class="text-right">Resolution</th>
-                                <th class="text-right no-sort">Entity address</th>
-                                <th class="d-none">URL</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            {% for sensor in asset.sensors: %}
-                            <tr>
-                                <td>
-                                    {{ sensor.name }}
-                                </td>
-                                <td>
-                                    {{ sensor.id }}
-                                </td>
-                                <td class="text-right">
-                                    {{ sensor.unit }}
-                                </td>
-                                <td class="text-right">
-                                    {{ sensor.event_resolution | naturalized_timedelta }}
-                                </td>
-                                <td class="text-right">
-                                    {{ sensor.entity_address }}
-                                </td>
-                                <td class="d-none">
-                                    /sensors/{{ sensor.id }}
-                                </td>
-                            </tr>
-                            {% endfor %}
-                        </tbody>
+                   
+                    <table class="table table-striped paginate nav-on-click" title="View data" id="sensorsTable">
+                        <div class="col-2 col-auto ms-auto">
+                            <select class="form-select form-select-sm" id="unitFilterOptions">
```

Comment:
As I said above, unit search is not needed here. It's needed  when we search sensors for config features like sensors to show. Save the code for that PR :)

Reply:
Apologies for that, I initially thought it would be useful for the front end in the future.